apps/datahub/src/environments/environment.ts

```diff
@@ -1,9 +1,14 @@
+import packageJson from '../../../../package.json'
```

Comment:
I think it would be a better practice to import only the version: `import { version } from '...'`

Otherwise the complete package.json content might end up in the bundle which might be a security issue.

Reply:
Hum, importing only the version does not seem to work. Following the discussion [here](https://stackoverflow.com/questions/34907682/how-to-display-the-app-version-in-angular) I decided to put the logic into the `environment.ts` to prevent exposing the entire `package.json` within the code. Checking the produced `main.js` in production this indeed seems to work, while in dev the `package.json` content is visible in the debugger.